playground/vitest.config.ts

```diff
@@ -2,10 +2,9 @@ import { defineConfigWithNuxt } from 'nuxt-vitest/config'
 
 export default defineConfigWithNuxt({
```

Comment:
Should we rename this method? I wonder if it's a bit confusing...

Reply:
It **is** with Nuxt (the config, the pipeline, etc.), just not the runtime environment by default. Do you have any better name in mind?

Reply:
That's true. And no, I don't 😆